### DISCUSSION TITLE
Is support for MONGODB-OIDC authentication mechanism planned?

### DISCUSSION THREAD

**Discussion by itxch** [prisma/prisma#25353] [Q&A] [answered]
Hi,

As the title states - is supporting MONGODB-OIDC authentication mechanism on the roadmap at all for Prisma?
I see that Prisma uses its own fork of the official rust driver - would we need to wait for support for that mechanism to be added by Mongo first?

This is their docs on it for node: https://www.mongodb.com/docs/drivers/node/current/fundamentals/authentication/enterprise-mechanisms/#std-label-node-mongodb-oidc-azure-imds

**Comment by ludralph** [comment, marked as answer]
Hi @itxch 👋 

This is currently not planned. However, you can please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that our engineering team could take a look.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
